analyzers/its/sources/CSharpLatest/CSharpLatest/CSharp10/NullableReferenceTypesExample.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace NonNullable
```

Comment:
This namespace, for some reason, was different than for other sources in the Net6 project. I can't see any valid reason, but I wonder if that was on purpose. Can you please double check?

Reply:
Good observation!
.NET 6 solution used to have 2 projects inside it, one of them had `<nullable>enabled...` and had only this file inside.
Since I moved it to the `CSharpLatest` solution, there is no need for a special namespace. 
And it's still nullable enabled, at the solution level.